game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -101,8 +108,19 @@ class Cell(val id: Int, player: Player, var position: Vector2 = new Vector2(0f,
     position.distanceTo(pos) <= radius
   }
 
-  def performAction(action: Action): Unit = {
+  def performAction(action: Action): Option[ScoreModification] = {
     target = action.target.toVector
+
+    val massToTrade = action.trade
```

Comment:
Actually, could we pick `min(action.trade, max(mass - Cell.MinMass, 0))` instead? Would fit with the [client documentation](https://github.com/DrPandemic/aigar.io/blob/master/clients/python/README.md#methods). We would need a test for the case where the cell is trying to trade too much (e.g. try to trade `100` mass when a cell has `30`, making sure that it gives only `10`).